naga/src/valid/expression.rs

```diff
@@ -1795,48 +1793,3 @@ fn f64_const_literals() {
     );
     assert!(result.is_ok());
 }
-
-/// Using I64 in a function's expression arena is forbidden.
-#[cfg(feature = "validate")]
-#[test]
-fn i64_runtime_literals() {
```

Comment:
Why are these tests being removed, as opposed to just un-fearure-gated?

Reply:
The answer is in the relevant commit message:

> chore(naga): remove broken Unsupported64Bit tests
> 
> Seemingly missed in https://github.com/gfx-rs/wgpu/commit/4e6f873da5e25acef9898f0f3490591c8be2e0b6.